tests/linearGaussian_snre_test.py

```diff
@@ -142,6 +143,8 @@ def test_c2st_sre_on_linearGaussian():
         (1, 1, "gaussian", "sre"),
         (2, 1, "uniform", "sre"),
         (2, 5, "gaussian", "aalr"),
+        (2, 1, "gaussian", "bnre"),
```

Comment:
these two `bnre` tests are failing locally (they are not executed during CI because they are marked "slow").

Reply:
I suggest to increase the number of simulations for `bnre`, it seems the posteriors are too broad / need more data. 

In particular, there seems to be a problem with the D_KL check at the end of the test. It's close to 0.0 for AALR, but above the threshold 0.15 for BNRE. Even when using 50k samples for BNRE the KL term is still above.

Reply:
To me it seems that there is something off with the calculation of the DKL in `test_utils.py`. Especially for `num_trials>1` it gives negative values. 
As the C2ST scores are below 0.6 for both BNRE tests I would be OK with excluding BNRE for the DKL check. 
I will create a separate issue for addressing the problem with the DKL calculation.

Reply:
I made some test with a higher simulation-budget and regularization_strength set to 50 instead of 100. On the 4 runs, bnre with 1 round succeeded on all, bnre with 5 rounds succeeded on 3 out of 4 and snre_b succeeded on 2 out of 4. So I think reducing the regularization_strength help but we can remove the DKL from the test.

Reply:
Sorry, for the delay in the responding! 
OK, I suggest we keep it like this so that some of the slow tests still fail on the DKL, which we will look into later.